packages/smart-vaults/src/utils/MultiSigner.sol

```diff
@@ -168,27 +169,24 @@ abstract contract MultiSigner {
 
         MultiSignerLib.MultiSignerStorage storage $ = _getMultiSignerStorage();
 
-        bytes memory signer;
-
         for (uint8 i; i < numSigners; i++) {
-            signer = signers_[i];
-            MultiSignerLib.validateSigner(signer);
-            $.signers[i] = signer;
+            MultiSignerLib.validateSigner(signers_[i]);
```

Comment:
don't think we need to validate

Reply:
actually we do have to prevent an empty signer being added because it can't be removed later

Reply:
ah true. btw probably cleaner to `use` `validate()` on `Signer[]` although... maybe we want to have that check inside `addSigner`? i'm not sure tbh

Reply:
yeah i feel like that should be inside `_addSigner`. shouldn't be possible to add an empty signer